go/engine/device_keygen.go

```diff
@@ -129,29 +125,33 @@ func (e *DeviceKeygen) Push(ctx *Context, pargs *DeviceKeygenPushArgs) error {
 
 	ds := []libkb.Delegator{}
 
-	if e.G().Env.GetEnableSharedDH() {
-		e.G().Log.CDebugf(ctx.NetContext, "DeviceKeygen#Push SDH:%v", e.G().Env.GetEnableSharedDH())
+	if e.G().Env.GetSupportPerUserKey() {
+		e.G().Log.CDebugf(ctx.NetContext, "DeviceKeygen#Push PUK:%v", e.G().Env.GetSupportPerUserKey())
 	}
 
-	var sdhBoxes = []keybase1.SharedDHSecretKeyBox{}
-	if e.G().Env.GetEnableSharedDH() && e.args.IsEldest {
+	var pukBoxes = []keybase1.PerUserKeyBox{}
+	if e.G().Env.GetSupportPerUserKey() && e.args.IsEldest {
```

Comment:
It is forced off for non-dev run-modes [here](https://github.com/keybase/client/pull/6939/files#diff-3ec785d6e391d57369aa99772ca02469R656). In dev it defaults to off and can be enabled with an env var or flag.